test/cases/parsing/template-string/cjs.js

```diff
@@ -41,5 +41,5 @@ it("should parse template strings in require.resolve", function() {
 
 	// Arbitrary assertion; can't use .ok() as it could be 0,
 	// can't use typeof as that depends on webpack config.
-	require.resolve(`./sync/${name}Test`).should.not.be.undefined;
+	require.resolve(`./sync/${name}Test`).should.not.be.undefined();
```

Comment:
One thing I just noticed -- you can't access `.should` on `undefined`, it's `TypeError`!

You're supposed to use `expect(value).to.not.be.undefined()`, but the test runner doesn't expose `expect`